modules/nf-core/gsea/gsea/main.nf

```diff
@@ -0,0 +1,77 @@
+process GSEA_GSEA {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::gsea=4.3.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gsea:4.3.2--hdfd78af_0':
+        'quay.io/biocontainers/gsea:4.3.2--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(gct), path(cls), path(gene_sets)
+
+    output:
+    tuple val(meta), path("*/*.rpt")                                                                                                        , emit: param_log
+    tuple val(meta), path("*/index.html")                                                                                                   , emit: index
```

Comment:
Ideally these should be emitted with channel names such as `rpt` and `index_html` (the latter to disabiduage from the `corr_plot_html` below